invenio_accounts/ext.py

```diff
@@ -93,6 +93,19 @@ def monkey_patch_flask_security():
             recoverable.hash_password = hash_password
             registerable.hash_password = hash_password
 
+        # Disable remember me cookie generation as it does not work with
+        # session activity tracking (a remember me token will bypass revoking
+        # of  a session).
+        def patch_do_nothing(*args, **kwargs):
+            pass
+        LoginManager._set_cookie = patch_do_nothing
```

Comment:
The cookie there is the `remember me` cookie.
I patched to be sure that it will never create that cookie.